pages/api/extension/setup.ts

```diff
@@ -0,0 +1,4 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
What is this supposed to do? 
Please at least add a comment here. If it is supposed to be empty, add a `no-op` comment here. If it is supposed to have something that is not implemented yet, add a `todo` comment.

Reply:
I have deleted this file both from git and from github ui. Don't know why this is still appearing here. Trying again

Reply:
Oh, this is an older commit, that's why. Can you please resolve this comment? I have removed this file, you can check it in the latest commit.